README.md

```diff
@@ -82,12 +82,19 @@ project:
 ## API
 
 The CRUD operations should be called from router.
-All storage replica sets should call `crud.init_storage()`
-(or enable the `crud-storage` role)
+
+All VShard storages should call `crud.init_storage()` after
```

Comment:
I think we also need to clarify router user grants.

If one is allowed to execute corresponding `crud` function, they don't need any other grants (for example, specific space read):

```lua
tarantool> box.schema.func.create('crud.insert')
---
...

tarantool> box.schema.user.create('myuser1', {password='mypass1'})
---
...

tarantool> box.schema.user.grant('myuser1', 'execute', 'function', 'crud.insert')
---
...

tarantool> c = net_box.connect('localhost:3301', {user='myuser1', password = 'mypass1'})
---
...

tarantool> c:call('crud.insert', {'customers', {1001, box.NULL, 'sdf', 3}})
---
- {'rows': [[1001, 882, 'sdf', 3]], 'metadata': [{'name': 'id', 'type': 'unsigned'},
    {'name': 'bucket_id', 'type': 'unsigned'}, {'name': 'name', 'type': 'string'},
    {'name': 'age', 'type': 'number'}]}
- null
...

```

Reply:
I've added a commit to README.md about `crud.init_storage()` and `crud.init_router()`.